src/components/MissionReservedList.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+const MissionReservedList = () => {
+  const listing = useSelector((state) => state.missionReducer);
+
+  const joinedMission = listing.filter((mission) => mission.join);
+
+  if (joinedMission.length === 0) {
+    return (
+      <div className="my-profile-item">
+        <h2>My Missions</h2>
+        <p>Be the first to join...</p>
+      </div>
+    );
+  }
```

Comment:
- [ ] [Highlight] & [Optional suggestion] Great work here adding a placeholder in case of no missions reserved. and it will be better if you used conditional rendering as `Inline If with Logical && Operator` or `Inline If-Else with Conditional Operator` see this [link](https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator) for more info.